src/lib/Service/MetaFieldType/MetaFieldDefinitionServiceInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Service\MetaFieldType;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Language;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinitionCreateStruct;
+use Ibexa\Contracts\Core\Repository\Values\ValueObject;
+
+interface MetaFieldDefinitionServiceInterface
```

Comment:
if it's not a contract, does it mean it's internal?

Reply:
Yes, it's internal